### PR TITLE
Modify the bootstrap script to exit on error, clean up

### PR DESCRIPTION
**Pull request by LukasVoda**
*Issue #, if available:* -

*Description of changes:*

When the bootstrapping or cleanup sequence fail we generally don't want to continue with the next
step because there are dependencies between them. I've turned the comments in the scripts to `echo`s to inform the user what is the current step.

Part of this commit is also a cleanup of the README and migration to AWS CLI v2.

Another issue I had was that the CLI is using a pager (less I think in my case) to display the API responses which blocks the script until you close the page. I've set the AWS_PAGER to an empty string to prevent this from happening and allow the script to finish without any user interaction.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -3,53 +3,55 @@
...
-sh install_chart.sh
-cd ..
-
-#Install test client requirements
-cd test_client
-pip install -r requirements.txt
-cd ..
\ No newline at end of file
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_PAGER=""
```
Comment (PapaPedro) [inline bootstrap.sh:7]: should we explicitly say we require AWS CLI 2?

File: cleanup.sh [hunk 2]
```diff
@@ -3,25 +3,19 @@
...
 
-sleep 300
+echo "Remove test deployment"
+(cd devopsguru_eks_test && sh uninstall_chart.sh)
 
-#Cleanup ECR repo
-cd ecr
-sh remove_repo.sh
-cd ..
+sleep 300
```
Comment (PapaPedro) [inline cleanup.sh:12]: Could we add a `echo "Sleep for 5 min..."` so that user knows what is going on

**Review by mirelap-amazon — APPROVED** [review]

**Review by PapaPedro — APPROVED** [review]

File: cleanup.sh (hunk 2 above)
Comment (LukasVoda) [reply cleanup.sh:12]: Good point, I will add that.

File: bootstrap.sh (hunk 1 above)
Comment (LukasVoda) [reply bootstrap.sh:7]: Good idea, I will make that more explicit in the README file in the requirements section.

**Pull request merged**